MeNext/Layout/PlayingScreen.cs

```diff
@@ -26,17 +26,14 @@ public PlayingScreen(MainController mainController)
                 Command = new Command(() => mainController.RequestPrevious())
             });
 
+            var playCommand = new Command(() =>
+            {
+                mainController.musicService.Playing = !mainController.musicService.Playing;
+            });
             layout.Children.Add(new Button
             {
                 Text = "Play/Pause",
-                Command = new Command((obj) =>
-                {
-                    if (mainController.musicService.Playing) {
-                        mainController.RequestPause();
-                    } else {
-                        mainController.RequestPlay();
-                    }
-                })
```

Comment:
Feature owner should write code to interface with backend. It should be *really* straight forwards for them. 

I'll write play/pause and volume controls tonight, those should be quick.